docs/account_creation.rst

```diff
@@ -16,7 +16,7 @@ and then creating the transaction which will charge the fee from the address.
 Deploying an account with DeployAccount transaction requires the following:
 
  - class_hash of the account contract
- - generating a private key and deployment salt
+ - private key and deployment salt
```

Comment:
note: Just to be precise, deploying an account requires  private key, not private key generation.